VSTSAgent/VSTSAgent.psm1

```diff
@@ -271,6 +277,7 @@ function Install-VSTSAgent {
     if ( $MinimumVersion ) { $findArgs['MinimumVersion'] = $MinimumVersion }
     if ( $MaximumVersion ) { $findArgs['MaximumVersion'] = $MaximumVersion }
     if ( $RequiredVersion ) { $findArgs['RequiredVersion'] = $RequiredVersion }
+    if ( $ProxyUrl ) {  $findArgs['ProxyUrl'] = $ProxyUrl }
```

Comment:
I don't think you want this here as Find-VSTSAgent doesn't support that arg and I think this will throw at runtime.

Reply:
At this point we're trying to find the agent to install.

Reply:
Should be ok now :)